### ISSUE TITLE
Long Property Names

### ISSUE THREAD

**Issue by philr-vistatec** [oasis-tcs/xliff-omos-jliff#51]
I hate to add this given where we are in the process but best before we publish. Could we revisit property names like `its_locQualityIssuesArray` and consider abbreviating to `its_lqiArray` and `its_locQualityIssueComment` to `its_lqiComment`?

**Comment by ysavourel** [comment]
+1

**Comment by philr-vistatec** [comment]
It is primarily aesthetic and secondarily byte size in a heavily annotated document.

**Comment by DavidFatDavidF** [comment]
To review in November 23 meeting and decide on path ahead..

**Comment by genivia-inc** [comment]
I agree with @DavidFatDavidF opinion, sharing a concern that shortening the property names will no longer correspond to XLIFF attributes, which may likely increase roundtrip complexity. Changing the property names will require a complete JSON Schema overhaul and extensive JLIFF Spec amendment effort.
